packages/catalog-search/src/SearchBox.jsx

```diff
@@ -11,6 +11,7 @@ import { SearchField } from '@edx/paragon';
 import debounce from 'lodash.debounce';
 import { connectSearchBox } from 'react-instantsearch-dom';
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This can likely be removed once `@edx/frontend-platform` is added back as a peer dependency.